frontend/src/app/main/ui/workspace/libraries.cljs

```diff
@@ -199,27 +197,66 @@
         empty-library? (empty-library? summary)
 
         selected       (h/use-shared-state mdc/colorpalette-selected-broadcast-key :recent)
+        dependencies   (mf/with-memo [shared-libraries]
+                         (into {} (map (fn [{:keys [id library-file-ids]}]
+                                         [id
+                                          (->> (str/split library-file-ids ",")
+                                               (map #(uuid/uuid %))
+                                               set)])
+                                       (vals shared-libraries))))
+
+        library-names  (mf/with-memo [shared-libraries]
+                         (into {} (map (fn [{:keys [id name]}]
+                                         [id name])
+                                       (vals shared-libraries))))
+
+        find-connected-to
+        (mf/use-fn
+         (mf/deps shared-libraries dependencies library-names)
+         (fn [library-id]
+           (->> dependencies
+                (filter (fn [[_ v]] (contains? v library-id)))
+                (map first))))
+
+        connected-to-linked?
+        (mf/use-fn
+         (mf/deps linked-libraries shared-libraries dependencies)
+         (fn [id]
+           (contains? linked-libraries id)))
```

Comment:
This can be already resolved with using `(contains? ...` directly where it is needed
This callback looks redundant and has more dependencies that are necessary i think

Reply:
Fixed